build.gradle

```diff
@@ -59,8 +59,18 @@ minecraft {
     }
 }
 
+repositories {
+    maven {
+        // Maven for CheatMode
+        name = "justAm0dd3r Maven"
+        url = "https://lookonthebrightsi.de/mc-mods/files/maven/"
+    }
+}
+
 dependencies {
-    minecraft 'net.minecraftforge:forge:1.15.2-31.1.0'
+    minecraft 'net.minecraftforge:forge:1.16.1-32.0.108'
+    // Directly open creative inventory from the survival mode - for testing purposes
+    runtimeOnly fg.deobf("justAm0dd3r.cheat-mode:cheat_mode-1.16.1:1.1.0")
```

Comment:
Remove the debug dependency from the PR then I am happy to merge!

Reply:
Of course, I only wanted to make testing it out a bit easier... removed it now 👍